os/kernel/Kconfig

```diff
@@ -508,6 +560,10 @@ config CANCELLATION_POINTS
 
 menu "Performance Monitoring"
 
+config SCHED_RESUMESCHEDULER
+	bool
+	default n
```

Comment:
Is this a sub menu of the `menu "Performance Monitoring"`?

Reply:
In nuttx resume and suspend is mainly used for performance actitivities. But there is some smp related code also in the resume api. So, I am moving this config to "Tasks and Scheduling" menu.